runtime/moonbase/src/weights/pallet_xcm.rs

```diff
@@ -119,21 +157,21 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
 	/// Proof: `ParachainSystem::PendingUpwardMessages` (`max_values`: Some(1), `max_size`: None, mode: `Measured`)
 	fn transfer_assets() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `217`
-		//  Estimated: `3682`
-		// Minimum execution time: 86_954_000 picoseconds.
-		Weight::from_parts(88_624_000, 3682)
-			.saturating_add(T::DbWeight::get().reads(7_u64))
-			.saturating_add(T::DbWeight::get().writes(3_u64))
+		//  Measured:  `12438`
+		//  Estimated: `20853`
+		// Minimum execution time: 344_458_000 picoseconds.
+		Weight::from_parts(352_779_000, 20853)
```

Comment:
We were using the old foreign assets, which were causing under-estimations on the pallet_xcm.

@TarekkMA, this is related to the previous improvement you were working on.